app/controllers/concerns/hmrc_temporary_ab_testable.rb

```diff
@@ -0,0 +1,39 @@
+module HmrcTemporaryAbTestable
+  CUSTOM_DIMENSION = 47 ## Not sure if this is correct
```

Comment:
I'm told it's correct for all of the tests to use 47, so we could remove this comment.